app/javascript/tests/public_health/query/AdvancedFilter.test.js

```diff
@@ -18,7 +18,9 @@ import {
   mockFilterAgeBetween,
   mockFilterManualContactAttemptsEqual,
   mockFilterManualContactAttemptsLessThan,
-  mockFilterEnrolledDateWithin,
+  mockFilterSymptomOnsetDateWithin,
```

Comment:
can we also update the tests in `patient_query_helper_test` to include the blank options?

Reply:
I decided not to do this for the date fields because there aren't any tests in there.  When I create my testing JIRAs later, I will be adding one for completing `patient_query_helper_test` because a lot of filters are missing.  I can add one for the blank labs though because those are already in there

Reply:
aren't the ones with the lab dates here? https://github.com/SaraAlert/SaraAlert/blob/master/test/helpers/patient_query_helper_test.rb#L247

there's no need to add additional tests to cover all the other advanced filters but since this ticket involves updating the one for labs, it would make sense to include/update tests for those (which shouldn't be too much because you already did some solid ground work on setting those up a while ago)

Reply:
yup that is exactly what I was saying.  I'll add some for labs now but the ones of type date (SO, LDE, latest report) will be added as a part of the JIRA addressing all the missing tests

Reply:
for reference, here is the JIRA: https://tracker.codev.mitre.org/browse/SARAALERT-1523

Reply:
all set: https://github.com/SaraAlert/SaraAlert/pull/961/commits/ef018fd75e0cee0dc2e043fb9cf200b15f7ea4b1